src/main/scala/edu/berkeley/ce/rockslicing/Joint.scala

```diff
@@ -204,12 +204,15 @@ object Joint {
   *        This avoids recalculation of a known dip angle.
   * @param dipDirectionParam An optional parameter that can be used to specify the dip direction for
   *        the joint. This avoids recalculation of a known dip direction.
+  * @param artificialJoint Parameter that identifies joint as being artificial joint introduced as part
+  *                        of load balancing
   */
 case class Joint(normalVec: (Double, Double, Double), localOrigin: (Double, Double, Double),
                  center: (Double, Double, Double), phi: Double, cohesion: Double,
-                 shape: Seq[((Double, Double, Double),Double)], dipAngleParam: Option[Double]=None,
+                 shape: List[((Double, Double, Double),Double)], dipAngleParam: Option[Double]=None,
```

Comment:
Yeah, I remember that. The reason for the change is related to Spark. If left as `Seq`, the compiler for some reason picks `Stream` which is non-serializable. Spark only works with serializable objects, so that's why I changed it. Even explicitly declaring `Joint` as `extends Serializable` doesn't fix the issue.